energyplus_regressions/tk_window.py

```diff
@@ -140,10 +142,27 @@ def __init__(self):
         Frame.__init__(self, self.root)
 
         # add the taskbar icon, but its having issues reading the png on Mac, not sure.
-        self.icon_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'ep.png')
-        if system() != 'Darwin':
-            img = PhotoImage(file=self.icon_path)
-            self.root.iconphoto(False, img)
+        if system() == 'Darwin':
+            self.icon_path = Path(__file__).resolve().parent / 'icons' / 'icon.icns'
+            if self.icon_path.exists():
+                self.root.iconbitmap(str(self.icon_path))
+            else:
+                print(f"Could not set icon for Mac, expecting to find it at {self.icon_path}")
+        elif system() == 'Windows':
+            self.icon_path = Path(__file__).resolve().parent / 'icons' / 'icon.png'
+            img = PhotoImage(file=str(self.icon_path))
+            if self.icon_path.exists():
+                self.root.iconphoto(False, img)
+            else:
+                print(f"Could not set icon for Windows, expecting to find it at {self.icon_path}")
+        else:  # Linux
+            self.icon_path = Path(__file__).resolve().parent / 'icons' / 'icon.png'
+            img = PhotoImage(file=str(self.icon_path))
+            if self.icon_path.exists():
+                self.root.iconphoto(False, img)
+            else:
+                print(f"Could not set icon for Windows, expecting to find it at {self.icon_path}")
+        fixup_taskbar_icon_on_windows(energyplus_regressions.NAME)
```

Comment:
Set the program icon in a platform-intelligent way, then use the PLAN-Tools functionality to update the taskbar icon on Windows.  (Could drop this last line into the Windows section, but I wasn't sure if one day this would expand to fixing up anything on other platforms.  Either way it's not hurting anything here.)